runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -505,8 +505,8 @@
 	ITEM_ANGUISH_ORNAMENT_KIT(ANGUISH_ORNAMENT_KIT, NECKLACE_OF_ANGUISH_OR),
 	ITEM_OCCULT_NECKLACE(OCCULT_NECKLACE, OCCULT_NECKLACE_OR),
 	ITEM_OCCULT_ORNAMENT_KIT(OCCULT_ORNAMENT_KIT, OCCULT_NECKLACE_OR),
-	ITE_AMULET_OF_FURY(AMULET_OF_FURY, AMULET_OF_FURY_OR),
-	ITE_FURY_ORNAMENT_KIT(FURY_ORNAMENT_KIT, AMULET_OF_FURY_OR),
+	ITEM_AMULET_OF_FURY(AMULET_OF_FURY, AMULET_OF_FURY_OR),
+	ITEM_FURY_ORNAMENT_KIT(FURY_ORNAMENT_KIT, AMULET_OF_FURY_OR),

```

Comment:
This'll likely get fixed upstream, we should let them deal with it.

Reply:
reverted